src/components/block.tsx

```diff
@@ -0,0 +1,54 @@
+// ----- Imports ----- //
+
+import React, { FC, ReactNode } from 'react';
+import { Option } from '@guardian/types/option';
+
+import { MaybeRender } from '../lib';
+
+
+// ----- Sub-Components ----- //
+
+const PublishDate = (props: { date: Date }) =>
+    <header>
+        <time>{props.date.toLocaleString()}</time>
+    </header>
+
+const UpdateDate = (props: { date: Date }) =>
+    <footer>
+        <time>{props.date.toLocaleString()}</time>
+    </footer>
+
+const Title = (props: { title: string }) =>
+    <h2>{props.title}</h2>
```

Comment:
Just personal preference, I personally don't think you need these sub components, I think its fine to put them in the block if they are only ever rendered there.

Reply:
Fair. Although I should say that I'm expecting these to become more complex once I start to apply styles to the `Block`. At the moment it's just barebones HTML to get storybook up and running.